build.sh

```
@@ -466,6 +458,20 @@ fi
 TESTS_DIR=${BUILD_DIR}/tests
 TEST_OUTPUT_FILE=${BUILD_DIR}/tests.out
 
+# Handle --clean directly: run the build tool's clean target against an existing
+# build directory and exit, without running ICU setup, configure, or a build.
+if [[ "${CMAKE_TARGET}" == "clean" ]]; then
+    if [ -d "${BUILD_DIR}" ]; then
+        printf "${BOLD_PINK}Cleaning${RESET} ${BUILD_DIR}\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)